<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App</string>
  </property>
  <widget class="QPushButton" name="close_server">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close App and Server</string>
   </property>
  </widget>
  <widget class="QComboBox" name="list_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select user</string>
   </property>
  </widget>
  <widget class="QComboBox" name="list_dirs">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirs for user</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_dir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove dir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_dir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add dir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add user</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bselect_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bselect_dir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start server</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
